### PR TITLE
Fix: widget/wp-search - Button and text input are not aligned (ED-2178)

### PR DESCRIPTION
**Pull request by nisan250**
Now when using the 'wp search' widget in hello theme, the button and the text input are aligned correctly.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	Stable tag: 2.3.1
 	Requires at least: 4.7
 	Tested up to: 5.6
+	Requires PHP: 5.6
```
Comment (StyleShit) [inline style.css:11]: Do we need that auto-generated string here?
@matipojo

**Review by StyleShit — APPROVED** [review]

**Review by Nevoss — APPROVED** [review]

File: assets/scss/theme/_general.scss [hunk 2]
```diff
@@ -84,3 +84,10 @@ to override any of the settings in this section, add your styling code in the cu
 .bypostauthor {
 	font-size: inherit;
 }
+
+// Specific style for 'wp search' widget: same alignment for the button and label elements.
+.elementor-widget-wp-widget-search {
+	.search-submit {
+		vertical-align: middle;
+	}
+}
```
Comment (matipojo) [inline assets/scss/theme/_general.scss:93]: Add a line at the end of the file.

File: style.css (hunk 1 above)
Comment (matipojo) [reply style.css:11]: No, remove it.

File: style.min.css [hunk 3]
```diff
@@ -8,6 +8,7 @@
 	Stable tag: 2.3.1
 	Requires at least: 4.7
 	Tested up to: 5.6
+	Requires PHP: 5.6
```
Comment (matipojo) [inline style.min.css:11]: Remove

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/scss/theme/_general.scss (hunk 2 above)
Comment (nisan250) [reply assets/scss/theme/_general.scss:93]: Done.

File: style.min.css (hunk 3 above)
Comment (nisan250) [reply style.min.css:11]: Done.

**Review by matipojo — CHANGES_REQUESTED** [review]
Remove unrelated code.

File: style.css (hunk 1 above)
Comment (nisan250) [reply style.css]: Done.

File: assets/scss/style.scss [hunk 4]
```diff
@@ -8,7 +8,6 @@
 	Stable tag: 2.3.1
 	Requires at least: 4.7
 	Tested up to: 5.6
-	Requires PHP: 5.6
```
Comment (matipojo) [inline assets/scss/style.scss:11]: Restore

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/scss/style.scss (hunk 4 above)
Comment (nisan250) [reply assets/scss/style.scss:11]: @matipojo I am not sure what to restore? I Removed the "Requires PHP: 5.6" since it was added in the last commit.

Comment (nisan250) [reply assets/scss/style.scss:11]: I have adde back the row: Requires PHP: 5.6

**Review by matipojo — APPROVED** [review]

**Pull request merged**